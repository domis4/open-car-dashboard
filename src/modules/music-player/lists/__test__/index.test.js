import React from 'react'
import { shallow } from 'enzyme'
import List from '../index'

describe('music-player list module index.js', () => {
  it('expects component to match snapshot', () => {
    const wrapper = shallow(<List />).dive()

    expect(wrapper).toMatchSnapshot()
  })
})
