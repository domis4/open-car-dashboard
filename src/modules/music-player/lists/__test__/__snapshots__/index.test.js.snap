// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`music-player list module index.js expects component to match snapshot 1`] = `
<WithStyles(Paper)
  className="View-root-01"
>
  <WithStyles(Typography)
    component="h1"
  >
    Playlists
  </WithStyles(Typography)>
  <WithStyles(Table)
    className="View-table-02"
  >
    <WithStyles(TableHead)>
      <WithStyles(TableRow)>
        <WithStyles(TableCell)>
          Title
        </WithStyles(TableCell)>
      </WithStyles(TableRow)>
    </WithStyles(TableHead)>
    <WithStyles(TableBody)>
      <WithStyles(TableRow)
        key="1"
      >
        <WithStyles(TableCell)
          component="th"
          scope="row"
        >
          Playlist 1
        </WithStyles(TableCell)>
      </WithStyles(TableRow)>
      <WithStyles(TableRow)
        key="2"
      >
        <WithStyles(TableCell)
          component="th"
          scope="row"
        >
          Playlist 2
        </WithStyles(TableCell)>
      </WithStyles(TableRow)>
      <WithStyles(TableRow)
        key="3"
      >
        <WithStyles(TableCell)
          component="th"
          scope="row"
        >
          Playlist 3
        </WithStyles(TableCell)>
      </WithStyles(TableRow)>
      <WithStyles(TableRow)
        key="4"
      >
        <WithStyles(TableCell)
          component="th"
          scope="row"
        >
          Playlist 4
        </WithStyles(TableCell)>
      </WithStyles(TableRow)>
      <WithStyles(TableRow)
        key="5"
      >
        <WithStyles(TableCell)
          component="th"
          scope="row"
        >
          Playlist 5
        </WithStyles(TableCell)>
      </WithStyles(TableRow)>
    </WithStyles(TableBody)>
  </WithStyles(Table)>
</WithStyles(Paper)>
`;
